src/components/Navbar/index.tsx

```diff
@@ -12,6 +15,25 @@ export const Navbar: React.FC = () => {
   return (
     <Wrapper>
       <InnerWrapper>
+        {false && (
```

Comment:
this will make it so that projects dropdown is not rendered until it's completely ready. Simply set to `true` if you wish to interact with it.

Reply:
i got rid of this. 1) because TS error and 2) because we don't really need it